.github/workflows/build.yml

```diff
@@ -1,5 +1,5 @@
 name: Gogstash
-on: [push, pull_request]
+on: [push, pull_request, workflow_dispatch]
```

Comment:
Why?

Reply:
add a manual trigger to execute the github action.

Reply:
When you need to trigger manually?

Reply:
test history version.